### ISSUE TITLE
Will jsk_visualization be ported to ROS2?

### ISSUE THREAD

**Issue by paulerikf** [jsk-ros-pkg/jsk_visualization#779]
Hey there, hope this is an o.k. place to ask this... any word on whether there will be a version of jsk visualization that will work with rviz2 / ros2?

Hard to imagine life without it <3

**Comment by k-okada** [comment]
Currently we have no plan/resource to port to ROS2, but volunteers are
welcome
--
◉ Kei Okada



2020年9月8日(火) 22:24 Paul <notifications@github.com>:

**Comment by fuhuan26** [comment]
Yeah, it will be great to have rviz2 compatible jsk_visualizaiton. The tool is so handy!

**Comment by arjunskumar** [comment]
any updates on this?

**Comment by ottojo** [comment]
Hi! We ported the text-overlay part to ROS2, and published the package here: https://github.com/teamspatzenhirn/overlay_rviz_plugins We intend to also port some of the other "overlay" plugins. The actually required code changes are rather minimal, and should all be contained in this commit, this might be helpful if someone decides to port the entire package: https://github.com/teamspatzenhirn/overlay_rviz_plugins/commit/5d52b380b67d0487ab321f34a9d31f261e47e685 (which we are not interested in currently)

**Comment by mqcmd196** [comment]
close via https://github.com/jsk-ros-pkg/jsk_visualization/pull/903
